src/components/pricing/PricingBoxes.tsx

```diff
@@ -41,9 +42,31 @@ const Styled = styled.div`
     }
 `
 
+const selfHostedPlans: PricingBoxProps[] = [
+    {
+        title: 'Free',
+        duration: 'For up to 5 users',
+        img: <object role="presentation" tabIndex={-1} data={Cloud} />,
+        features: ['Unlimited Use', 'Private & Public Repos', 'Admin dashboard'],
+        price: 'Free',
+        btnText: 'Install Now',
+        btnBackground: true,
+        link: '/self-hosted/#install'
+    },
+    {
+        title: 'Teams',
```

Comment:
How about calling it Unlimited instead of Teams?

```suggestion
        title: 'Unlimited',
```